pkg/operator/status.go

```diff
@@ -289,9 +291,79 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if we didn't run
+	if supported, v := optr.isKubeletSkewSupported(); !supported {
+		coStatus.Reason = "KubeletSkewUnsupported"
+		coStatus.Message = fmt.Sprintf("machine config pool contains nodes with an unsupported kubelet version skew. Please see `oc get nodes` for details and upgrade all nodes so that they have a kubelet Version of at least %v.", v)
```

Comment:
`upgrade all nodes so that they have a kubelet Version of at least %v.", v`

If v is the apiserver version, the nodes don't have to be updated to that version but to that version-2 at a minimum no?

Reply:
> `upgrade all nodes so that they have a kubelet Version of at least %v.", v`
> 
> If v is the apiserver version, the nodes don't have to be updated to that version but to that version-2 at a minimum no?

The version skew does not support version - 2 version number. It should be the version with n-1 (n is the minor version of the version). If we want to report the version with a specific version number, will need another parsing function to do the calculation. https://github.com/openshift/machine-config-operator/pull/2658/files#diff-8a6a064b4db19eb3b29e5280447ac3ba9496e847391489d25c1282767c3354c5R371
Do you think it worth adding this func?

Reply:
I think your update covers it by printing minimalsupported version. :+1: